sys/net/link_layer/l2util/l2util.c

```diff
@@ -118,6 +128,12 @@ int l2util_ipv6_iid_from_addr(int dev_type,
                 return -EINVAL;
             }
 #endif  /* defined(MODULE_CC110X) || defined(MODULE_NRFMIN) */
+#if defined(MODULE_NIMBLE_NETIF)
+        case NETDEV_TYPE_BLE:
+            l2util_eui64_from_addr(dev_type, addr, addr_len, iid);
+            /* for BLE we don't flip the universal local flag... */
```

Comment:
```suggestion
            /* for BLE we don't flip the universal/local flag... */
```

Otherwise the term is a bit self-contradicting ;-)

Reply:
fixed :-)